### ISSUE TITLE
Add list of minimal features needed on single end and paired end predict upload pages

### ISSUE THREAD

**Issue by mahafarhat** [farhat-lab/gentb-site#240]
@mgro is preparing a new table that includes WDNN features and RF features. 

We should place it here for both single-end and paired-end pages: 
![image](https://user-images.githubusercontent.com/8130645/118345237-1a8d6180-b501-11eb-8109-bae75b68de69.png)

**Comment by doctormo** [comment]
@mgro Let me know when the table is ready to add.

**Comment by mgro** [comment]
Hi @doctormo - here is the table as word doc or PDF.
[table_for_website.docx](https://github.com/farhat-lab/gentb-site/files/6503207/table_for_website.docx)
<img width="634" alt="table_website" src="https://user-images.githubusercontent.com/22810016/118697903-ce327200-b7dd-11eb-8b77-d282760536e3.png">

**Comment by mahafarhat** [comment]
@doctormo when you update the single end upload page with this table, can you please add a sentence describing the naming convention for the single end fastq file? I believe we require the fastq file to have .fastq or .fastq.gz   at th emoment

We just had a user try to upload fastq files labeled .fq or .fq.gz. I think its fine to accept this naming convention too. So in sum
1- please allow .fq and .fq.gz files
2- Provide the user with instruction saying files should be labeled .fastq or .fq and gzipped files are accepted and preferred.

**Comment by doctormo** [comment]
Maha can you provide the completed sentence required for these three requests?

**Comment by doctormo** [comment]
I've added the "fq" and "fq.gz" uploading support, this doesn't mean it will work in the pipelines though.

**Comment by mahafarhat** [comment]
For complete sentence for single end upload instructions please use the following:
"
Create a prediction from single-ended FastQ genetic sequence. You can batch upload multiple FastQ files.

Both compressed and uncompressed files are accepted but the former are preferred.
Please label paired end FastQ files using either the .fastq or .fq extension. No other extensions are accepted. 
"

**Comment by mahafarhat** [comment]
for the fq support, make sure it accepts it the same way as a .fastq but then renames it as .fastq so we don't get any downstream errors.

**Comment by doctormo** [comment]
All these issues should now be fixed.

 * I've made new templates in `apps/predict/templates/predict/types/` which should allow you to edit and add information to any of the predict types without having to edit python.
 * This included fixing both fastq-pair and manual templates that had their own hack additions. Now all standard.
 * The doc variable is now only used for the selection screen, so this description can be different. It's still in the forms.py
 * FQ files are renamed to FastQ when the pipeline is submitted, but not before.
 * Various other small fixes found as a I ran the test suites.
